README.md

```diff
@@ -122,32 +154,37 @@ as you maybe know, we save your mock trip and the state of that. for exporting t
-
-2. **file_owner(String)**: We save the User UUID in this field.
-
-3. **version_code(Int)**: We save the application version code in this field.
+|Name| Type | Description |
+|--|--|--|
+| **file_created_at** | String | Creation Date/Time of the exported JSON file
+|**file_owner**|String|User UUID
+|**version_code**|Number| Application version code
+|**mock_information**|Object| Necessary mock data
+|**route_information**|Object| Route information and line. **route_lines** is the most important field in this object. It is an array containing a list of positions (nodes) to draw lines on the map. Every array item has a **latitude** and **longitude** for its position.
 
-4. **mock_information(Object)**: For saving mock information, we use an object that we call **mock_information**. as you can see, we save necessary information in that.
 
-5. **route_information(Object)**: For saving route information and lines, we use this object for it. the important field of this object is **route_lines**. this object contains a list of positions that we should have for drawing lines on the map. every line has a **latitude** and **longitude** for its position.
+### Import
 
-we save this file in the format of `.json`. you can share it everywhere you want. maybe you think how can we import it?
+As you see in the previous section, you can export and share your mock data with your friends and colleagues in JSON format. To import any data to your project, you can easily import this file using the import feature. ( We have plans in the near future to add other methods to import data )
 
-### Importing structure
+We store this imported JSON data like standard mock data in a different database table. We add the following additional column to each row:
+- fileCreatedAt
+- fileOwner
+- versionCode
 
-as you can see in exporting section, you can share a `.json` file to your friends everywhere you want. for importing it, we can **Only** receive this `.json` file in the import section of the application(for the future, We have a plan for importing mocks in many ways!).
+We separated the imported data so we could manage them easily. These are the reasons that give us the ability to do so:
 
-We save this `Json` file information like normal mocks but in a different table in the application database. We also save some extra information like: `fileCreatedAt`, `fileOwner` and `versionCode` in imported mock table. the reason that We split these mocks from normal mocks that you create in the application is, maybe the information of imported mocks can be less than normal mocks in the future. also, We have a plan for a new UI to show imported mocks in a different place in the application. so we create a new place for saving these mocks to managing them very easily in the future.
+- We may have less imported data in the future.
+- We have a plan to implement a separate UI to display imported data.
```

Comment:
Let me replace and improve this section with the following sentences:

- Maybe We have a different structure for the imported mocks in the future
- In the future, We wanna implement a different flow for showing imported mocks in UI and Logic too.

Reply:
```suggestion
- Maybe We have a different structure for the imported mocks in the future
- In the future, We wanna implement a different flow for showing imported mocks in UI and Logic too.
```